lib/internal/modules/esm/get_source.js

```diff
@@ -40,6 +40,6 @@ async function defaultGetSource(url, { format } = {}, defaultGetSource) {
   if (policy?.manifest) {
     policy.manifest.assertIntegrity(parsed, source);
   }
-  return { source };
+  return source;
 }
 exports.defaultGetSource = defaultGetSource;
```

Comment:
Likewise, maybe we should rename this to `getSource`.

Reply:
same reason (not married to it)